addons/certificate/tests/test_key.py

```diff
@@ -0,0 +1,47 @@
+import base64
+
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import ec, rsa
+
+from odoo.tests import TransactionCase, tagged
+from odoo.tools import file_open
+
+
+@tagged('post_install', '-at_install')
+class TestKey(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        private_key = rsa.generate_private_key(public_exponent=65537, key_size=2048)
+        cls.test_key = cls.env['certificate.key'].create({
```

Comment:
since you create also need keys in test_certificate, maybe you should merge both files together